DependencyInjection/FOSRestExtension.php

```diff
@@ -37,6 +40,29 @@ public function prepend(ContainerBuilder $container)
         if ($config['view']['view_response_listener']['enabled']) {
             $container->prependExtensionConfig('sensio_framework_extra', ['view' => ['annotations' => false]]);
         }
+
+        $bundles = $container->getParameter('kernel.bundles');
+        if (isset($bundles['WebProfilerBundle'])) {
+            $profilerConfigs = $container->getExtensionConfig('web_profiler');
```

Comment:
That's right. I don't think this part of their code will change but in the case it does, it may be better to use a container parameter (https://github.com/FriendsOfSymfony/FOSRestBundle/compare/master...Ener-Getick:aafa3a7ce39e316e263fc5a5b88a344709b1238b).

We have to decide what's the best to do ...